src/envoy/utils/filter_state_utils.cc

```diff
@@ -25,13 +27,20 @@ using ::Envoy::Router::StringAccessor;
 using ::Envoy::Router::StringAccessorImpl;
 using ::Envoy::StreamInfo::FilterState;
 
+// FilterState container needed to store the google.rpc.Status error proto.
+struct RpcStatusWrapper : public Envoy::StreamInfo::FilterState::Object {
+  google::rpc::Status status_;
+
+  Envoy::ProtobufTypes::MessagePtr serializeAsProto() const override {
```

Comment:
I don't think we need this. It is only used for access_log to write filerState

Reply:
Well we need to keep the object, so may as well keep this function in case a user configured the access log format. It doesn't add any complexity